org.eclipse.m2e.launching/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.eclipse.m2e.launching;singleton:=true
-Bundle-Version: 2.0.500.qualifier
+Bundle-Version: 2.0.501.qualifier
```

Comment:
```suggestion
Bundle-Version: 2.0.600.qualifier
```
By default the service/micro version should be bump by `+100`, just in case we have to do a in between maintenance release later (we have not yet done it, but since that is cheap why not be prepared, just in case).